### DISCUSSION TITLE
prime vue Carousel cannot work. release 3.10.0

### DISCUSSION THREAD

**Discussion by Yootthanaboon** [primefaces/community#1544] [PrimeVue] [answered]
Carousel cannot work.
<img width="1673" alt="Screenshot 2567-03-28 at 21 07 13" src="https://github.com/primefaces/community/assets/159749002/53ceb56d-41dd-490f-b6ce-d4a1e3c2a815">

**Comment by tugcekucukoglu** [comment, marked as answer]
It was fixed. Could you check again?
